CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ librdkafka.
    for situations where the existing `rd_kafka_producev()` va-arg approach
    can't be used.
  * Added RTT/delay simulation to mock brokers.
+ * Added `sticky.linger.ms` to improve batching of null-keyed messages
```

Comment:
Sticky is a bit ambigious, especially with the sticky assignor being added, so suggest something like `sticky.random.partitioner.linger.ms`